{"name": "Проверка ошибки принятия заказа без id курьера", "status": "passed", "start": 1720188154260, "stop": 1720188154694, "uuid": "6413deb0-6bd8-4090-a006-fe500f098ccc", "historyId": "5c1e0c1c9af910be02b14550c5eebcc4", "testCaseId": "5c1e0c1c9af910be02b14550c5eebcc4", "fullName": "tests.test_accept_order.test_accept_order_fail.TestAcceptOrderFail#test_accept_order_without_courier_id", "labels": [{"name": "parentSuite", "value": "tests.test_accept_order"}, {"name": "suite", "value": "test_accept_order_fail"}, {"name": "subSuite", "value": "TestAcceptOrderFail"}, {"name": "host", "value": "DESKTOP-Q9FKTCA"}, {"name": "thread", "value": "11792-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order.test_accept_order_fail"}]}